osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -91,11 +94,17 @@ private double strainValueOf(DifficultyHitObject current)
                     wideAngleBonus *= angleBonus * (1 - Math.Min(wideAngleBonus, Math.Pow(calcWideAngleBonus(lastAngle), 3))); // Penalize wide angles if they're repeated, reducing the penalty as the lastAngle gets more acute.
                     acuteAngleBonus *= 0.5 + 0.5 * (1 - Math.Min(acuteAngleBonus, Math.Pow(calcAcuteAngleBonus(lastLastAngle), 3))); // Penalize acute angles if they're repeated, reducing the penalty as the lastLastAngle gets more obtuse.
 
-                    angleBonus = acuteAngleBonus * acute_angle_multiplier + wideAngleBonus * wide_angle_multiplier; // add the angle buffs together.
+                    angleBonus = Math.Max(acuteAngleBonus * acute_angle_multiplier, wideAngleBonus * wide_angle_multiplier); // Take the max of the multipliers.
```

Comment:
This seems like something that belongs to base PR. Is this intentional change?

Reply:
This is an intentional change, I made note of it in this PR. It was something that I felt made sense and probably should have been in the base PR.

> Note that I did add an adjustment to the base PR that takes the max instead of the combined angleBonus for both acute and wide angles. This was done as there were some maps taking advantage of this that probably shouldn't have been buffed so hard. It also doesn't really make sense to double dip anglebuff in that way.

Reply:
Sure, makes sense